plugins/modules/aci_l4l7_device_selection_interface_context.py

```diff
@@ -0,0 +1,449 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# Copyright: (c) 2025, Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_device_selection_interface_context
+short_description: Manage L4-L7 Device Selection Policy Logical Interface Contexts (vns:LIfCtx)
+description:
+- Manage L4-L7 Device Selection Policy Logical Interface Contexts
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  contract:
+    description:
+    - The name of an existing contract.
+    type: str
+    aliases: [ contract_name ]
+  graph:
+    description:
+    - The name of an existing Service Graph Template.
+    type: str
+    aliases: [ service_graph, service_graph_name ]
+  node:
+    description:
+    - The name of an existing Service Graph Node.
+    type: str
+    aliases: [ node_name ]
+  context:
+    description:
+    - The name of the logical interface context.
+    type: str
+  l3_destination:
+    description:
+    - Whether the context is a Layer3 destination.
+    - The APIC defaults to C(true) when unset during creation.
+    type: bool
+    aliases: [ l3_dest ]
+  permit_log:
+    description:
+    - Whether to log permitted traffic.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  bridge_domain:
+    description:
+    - The Bridge Domain to bind to the Context.
+    type: str
+    aliases: [ bd, bd_name ]
+  bridge_domain_tenant:
+    description:
+    - The tenant the Bridge Domain resides in.
+    - Omit this variable if both context and Bridge Domain are in the same tenant.
+    - Intended use case is for when the Bridge Domain is in the common tenant, but the context is not.
+    type: str
+    aliases: [ bd_tenant ]
+  logical_device:
+    description:
+    - The Logical Device to bind the context to.
+    type: str
+  logical_interface:
+    description:
+    - The Logical Interface to bind the context to.
+    type: str
+  redirect_policy:
+    description:
+    - The Redirect Policy to bind the context to.
+    type: str
+  permit_handoff:
+    description:
+    - Indicates whether to allow handoff of traffic to the associated logical interface.
```

Comment:
Does APIC default these Booleans to anything when unset? `permit_handoff`, `acl`, `rule_type`?